content/tutorials/aws-persistence.textile

```diff
@@ -0,0 +1,241 @@
+# Select @IAM@ (you can search in the toolbar if necessary)
+# Click @Users@ from the left hand menu
+# Click @Add users@
+# Set the @User name@ as @ably-persistence@
+# Check the @Programmatic access@ checkbox
+# Click @Next: permissions@
+# On the @Set permissions@ page:
+## Click @Attach existing policies directly@
+## Type @AWSLambdaRole@ into the search box
+## Check the @AWSLambdaRole@ checkbox
+## Click @Next: Review@
+# Click @Create user@, ignoring the optional tag options
+# Your new user will be displayed with its @Access key ID@ and @Secret access key@:
+## Copy the access key ID
+## Click the @Show@ link in the last column to reveal the secret key
+## Copy the secret key into your text file
+
+h2(#set-up-s3). Set up S3
+
+For this tutorial we will be using S3 to store and update a simple plain text file. S3 is one of many scalable storage solutions in the AWS ecosystem but you can apply the basic principles to other services and other file types.
+
+The first step is to set up our S3 bucket to keep the file we'll be using for persisting in. From the "AWS dashboard":http://console.aws.amazon.com/:
+
+# Click @Services@ from the top left menu
+# Select @S3@ (you can search in the toolbar if necessary)
+# Click @Create bucket@
+# Set the @Bucket name@ to a unique name: @your-bucket-name-here@
+# Click @Create bucket@
+
+While we're in S3 we will also create the file we'll be updating with our persisted data:
+
+# In your local environment create an empty @test.txt@ file
+# Click @your-bucket-name-here@
+# Click @Upload@
+# Click @Add files@
+# Navigate to and select your @test.txt@ file then click @Open@
+# Click @Upload@
+
+h2(#role). Set up Role
+
+Before we can create the Lambda function itself we also need to create a Role it can be assigned to determine its access levels. As mentioned previously this is a tutorial so we're going to provide high level access, but you may want to employ stricter permissions in a production deployment.
+
+
+From the "AWS dashboard":http://console.aws.amazon.com/:
+
+# Click @Services@ from the top left menu
+# Select @IAM@ (you can search in the toolbar if necessary)
+# Click @Roles@ from the left hand menu
+# Click @Create role@ button
+# Select the @AWS Service@ tile
+# Click @Lambda@ as the @Common use cases@
+# Click @Next@
+# Chose the correct roles:
+## Type @AWSLambdaBasicExecutionRole@ into the search box
+## Check the @AWSLambdaBasicExecutionRole@ checkbox
+## Click @Clear filters@
+## Type @AmazonS3FullAccess@ into the search box
+## Check the @AmazonS3FullAccess@ checkbox
+## Click @Next@
+# Enter @ably-persistence@ as the @Role Name@
+# Click @Create role@
+
+h2(#create-lambda). Create the Lambda function
+
+We can now create the Lambda role itself. Lambda functions as the logic center and pipeline for persistence in this tutorial, you can set up more comprehensive functions than we will here to optimize the persistence to suite your business needs.
```

Comment:
Because a Lambda IS a function, using functions as a verb here might cause confusion.